server/datastore/mysql/migrations/tables/20240424124712_AddSoftwareInstallerTables.go

```diff
@@ -41,7 +41,7 @@ CREATE TABLE IF NOT EXISTS software_installers (
   post_install_script_content_id int(10) unsigned DEFAULT NULL,
 
   -- used to track the ID retrieved from the storage containing the installer bytes
-  storage_id binary(64) NOT NULL,
+  storage_id varchar(64) COLLATE utf8mb4_unicode_ci NOT NULL,
```

Comment:
Heads-up folks, yet another DB migration change - initially we thought we'd store the raw (binary) hash of the installer here, but we've ended up storing the hex-encoded (plain ascii string) version, so changing to `varchar` will avoid issues such that a shorter-than-64 bytes `binary(64)` gets returned by mysql padded with null bytes (`0x00`). Again, this will mean a local DB reset :) @ghernandez345 @dantecatalfamo @roperzh @gillespi314 @jahzielv